paper/mitchell-geophysics.tex

```diff
@@ -0,0 +1,1999 @@
+%%   numbers-  selects numerical citations
+%%   super  -  numerical citations as superscripts
+%%   sort   -  sorts multiple citations according to order in ref. list
+%%   sort&compress   -  like sort, but also compresses numerical citations
+%%   compress - compresses without sorting
+%%
+%% \biboptions{comma,round}
+
+% \biboptions{}
+
+\usepackage[T1]{fontenc}
+\usepackage[utf8]{inputenc} % The default since 2018
+\usepackage{amsmath}
+% \usepackage{breqn}
+% \ExplSyntaxOn
+% \cs_set_eq:NN \intexpr_eval:w \int_eval:w
+% \cs_set_eq:NN \intexpr_eval_end: \int_eval_end:
+% \ExplSyntaxOff
+
+\newcommand{\midtilde}{\raisebox{-0.25\baselineskip}{\textasciitilde}}
+\newcommand{\IcelandicDh}{\textnormal{ð}}
+
+\usepackage{color}
+\usepackage{color, colortbl}
+\definecolor{greytext}{gray}{0.5}
+
+\usepackage{enumitem}
+\usepackage{boldline}
+
+
+\usepackage{subcaption}
+\usepackage{changepage}
+\usepackage{rotating}
+\usepackage{lscape}
+
+\journal{Journal of Applied Geophysics}
+
+\begin{document}
+
+\begin{frontmatter}
+
+%% Title, authors and addresses
+
+%% use the tnoteref command within \title for footnotes;
+%% use the tnotetext command for the associated footnote;
+%% use the fnref command within \author or \address for footnotes;
+%% use the fntext command for the associated footnote;
+%% use the corref command within \author for corresponding author footnotes;
+%% use the cortext command for the associated footnote;
+%% use the ead command for the email address,
+%% and the form \ead[url] for the home page:
+%%
+%% \title{Title\tnoteref{label1}}
+%% \tnotetext[label1]{}
+%% \author{Name\corref{cor1}\fnref{label2}}
+%% \ead{email address}
+%% \ead[url]{home page}
+%% \fntext[label2]{}
+%% \cortext[cor1]{}
+%% \address{Address\fnref{label3}}
+%% \fntext[label3]{}
+
+% \title{DC Resistivity Survey Design Enhancement to Resolve Conductivity Structures Around Tunnels or Mine Workings}
+\title{Using DC Resistivity Ring Array Surveys to Resolve Conductive Structures Around Tunnels or Mine-Workings}
+% \title{Using a DC Resistivity Survey Comprised of an Ensemble of Ring Arrays to Resolve Conductive Structures Around Tunnels or Mine-Workings}
+
+%% use optional labels to link authors explicitly to addresses:
+%% \author[label1,label2]{<author name>}
+%% \address[label1]{<address>}
+%% \address[label2]{<address>}
+
+% \author[1,2]{Michael A. Mitchell\corref{mycorrespondingauthor}}
+% \cortext[mycorrespondingauthor]{Corresponding author}
+% \ead{mamitchell@usgs.gov}
+
+% \author[1]{Douglas W. Oldenburg}
+
+% \address[1]{Geophysical Inversion Facility (GIF), \\
+% Department of Earth, Ocean and Atmospheric Sciences, \\
+% University of British Columbia, \\
+% 2020-2207 Main Mall, \\
+% Vancouver, British Columbia \\
+% Canada, V6T 1Z4}
+
+% \address[2]{U.S. Geological Survey, \\
+% Volcano Science Center, \\
+% California Volcano Observatory, \\
+% 350 N. Akron Rd., \\
+% Moffett Field, CA 94035}
+
+
+\begin{abstract}

```

Comment:
@ fwkoch, whenever you have a moment, I would appreciate your help on this one! I suspect I am missing something obvious -- the abstract doesn't seem to show up in the myst preview